src/lib/charts.ts

```diff
@@ -23,10 +20,7 @@ type PriceAtDate = (args: PriceAtDateArgs) => number
 
 export const priceAtDate: PriceAtDate = ({ date, priceHistoryData }): number => {
   const { length } = priceHistoryData
-  if (!length) {
-    moduleLogger.warn('priceAtDate: no price history data - something will be wrong!')
-    return 0
-  }
+  if (!length) return 0
```

Comment:
with new animated charts - we don't need to wait on all the market data, so this was creating low spew